lib/graphql/schema/member/has_path.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module GraphQL
+  class Schema
+    class Member
+      module HasPath
+        # @return [String] A description of this member's place in the GraphQL schema
+        def path
```

Comment:
Can this be memoized?

Reply:
🤔  Maybe, probably, but I'm not that sure. Right now, we don't _force_ anything to be frozen, this would be one of the first things in the class-based API to be cached. Do you think it will be a performance bottleneck?